tests/integration_tests/test_hyperopt_ray_horovod.py

```diff
@@ -137,9 +138,9 @@ def on_hyperopt_start(self, *args, **kwargs):
 
 
 @contextlib.contextmanager
-def ray_start_4_cpus():
+def ray_start_10_cpus():
     res = ray.init(
-        num_cpus=4,
+        num_cpus=10,
```

Comment:
I don't think GH runners have 10 cpus. Do we really need this many? I think if it can't run with like 6 at most, there's probably a bug somewhere.

Reply:
@tgaddair we would need at least one more CPU than the CPUs used in hyperopt trials (discussed in this [slack thread](https://predibase.slack.com/archives/C02J6QF4T0A/p1652751752635729)).

We could artificially restrict hyperopt to run less than 6 trials so that there are CPUs available for dask computation by setting `max_concurrent_trials` to something less than 6. For now, I've changed the number of CPUs to 7.